packages/amplify_datastore/android/src/main/kotlin/com/amazonaws/amplify/amplify_datastore/FlutterModelProvider.kt

```diff
@@ -19,14 +19,14 @@ import com.amplifyframework.core.model.Model
 import com.amplifyframework.core.model.ModelProvider
 import com.amplifyframework.core.model.ModelSchema
 import com.amplifyframework.util.Immutable
+import java.util.Collections
 
 class FlutterModelProvider private constructor() : ModelProvider {
     private val modelSchemaMap: MutableMap<String, ModelSchema> =
             HashMap()
 
-    // TODO change it to return an empty set once it's properly supported in the amplify-android library
     override fun models(): Set<Class<out Model?>>? {
-        return java.util.Collections.emptySet()
+        return Collections.emptySet()
     }
 
     override fun version(): String {
```

Comment:
Now sending model provider version from dart to Kotlin. 

Also changed the way that config works for the Datastore.  We send a ModelProviderInterface instead of the List<ModelSchema> 

As the config method signature has changed, @Amplifiyer you might want to look at those changes too (it's in a separate commit labeled "Send ModelProviderVersion".